### DISCUSSION TITLE
95% emitting emit phase takes much longer in Node >=16

### DISCUSSION THREAD

**Discussion by GaganpreetKaurKalsi** [webpack/webpack#18213] [Q&A]
# Bug report

**What is the current behavior?**

In development mode, 

Currently we are running our application on `Node 14` and it takes around `30 secs` for the `95% emitting emit` phase to complete.

After upgrading to Node 16 and higher, it takes around `~2mins` to get through the `95% emitting emit` phase ( no change in webpack configuration).

**If the current behavior is a bug, please provide the steps to reproduce.**

This should be observed in any large scale project using webpack.

**What is the expected behavior?**

Upgrading node version shouldn't impact the compile time this much.

**Other relevant information:**
webpack version: 5.90.1
Node.js version: 14.20.0
Operating System: MacOS
Additional tools:

**Comment by alexander-akait** [comment]
Please create reproducible test repo or profile, without configuration I can't help, sorry
